svea_vision/.gitignore

```diff
@@ -1 +1,3 @@
-**/__pycache__
\ No newline at end of file
+**/__pycache__
+*.pt
+images/*
```

Comment:
When is `images` used?

Reply:
It was used for `static_image_publisher` node. It could be removed.